documentation/paths/user.js

```diff
@@ -55,8 +37,23 @@ module.exports = {
                 $ref: '#/components/schemas/Error'
               },
               example: {
-                message: 'User´s email already exists',
-                internal_code: 'invalid_parameters'
+                message: 'This field is required password',
+                internal_code: 'missing_data_error'
+              }
+            }
+          }
+        },
+        422: {
+          description: 'Unprocessable Entity',
+          content: {
+            'application/json': {
+              schema: {
+                $ref: '#/components/schemas/Error'
+              },
+              example: {
+                message:
```

Comment:
I would leave something more generic here

     
 `example: {`
                `internal_code: 'validation_error',`
                `message: 'Some params do not have a valid value'`
             ` }`